src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -158,6 +158,11 @@ private static string GetCorrectCasedPath(string path)
                         // This handles short path names
                         exactPath += StringLiterals.DefaultPathSeparator + item;
                     }
+                    else if (item.IndexOfAny(new char[] { '*', '?' }) >= 0)
```

Comment:
We can re-use Utils.Separators.StarOrQuestion static value.

Reply:
Excellent; I was wondering if there was a static value somewhere.  Thanks!